### ISSUE TITLE
Warning by pub "updated to the v2 Android Plugin APIs"

### ISSUE THREAD

**Issue by dhorajiaabhishek** [fweinaug/system_settings#10]
The plugin `system_settings` is built using an older version of the Android plugin API which assumes that it's running in a full-Flutter environment. It may have undefined behaviors when Flutter is integrated into an existing app as a module.
The plugin can be updated to the v2 Android Plugin APIs by following https://flutter.dev/go/android-plugin-migration.

Can you update this.

**Comment by efraespada** [comment]
```
The plugins `system_settings` use a deprecated version of the Android embedding.

To avoid unexpected runtime failures, or future build failures, try to see if these plugins support the Android V2 embedding. Otherwise, consider removing them since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.
```

**Comment by parmeetmaster** [comment]
The plugin `system_settings` uses a deprecated version of the Android embedding.
To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.
If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.

**Comment by TrifiAmanallah** [comment]
Is there someone working to fix this ?

**Comment by Kang-Kim1** [comment]
Any updates?

**Comment by raphaaugustosilva** [comment]
Any updates?

**Comment by byazit** [comment]
Anyone, please fix this!

**Comment by lp0int** [comment]
> Anyone, please fix this!

Hi, you can use this commit [https://github.com/fweinaug/system_settings/pull/18](https://github.com/fweinaug/system_settings/pull/18)

**Comment by mikeborodin** [comment]
hi, I see there's open PR for this issue, is there some blocker? Can I help somehow?

**Comment by darkstarx** [comment]
@fweinaug what is the reason to not fix this issue? If you have no time, please add someone else to the repo to fix this problem and publish a new version of the package.

**Comment by fweinaug** [comment]
I just released version 2.1 which contains this change. thanks everyone and sorry it took so long!
